devnet/README.md

```diff
@@ -23,6 +23,39 @@ docker-compose down
 
 > **_NOTE:_**  The devnet is not stateful, so subsequent runs will start from the genesis block.
 
+## Running Cross Client Devnet
+
+A cross-client devnet is network of gossamer node(s) and substrate node(s).
+
+Steps to run a two node cross client devnet
+
+- Create `genesis-spec.json`
+```
+./polkadot build-spec --disable-default-bootnode --dev > genesis-spec.json
+```
+
+- Edit `genesis-spec.json` as per your needs. Add extra authorities if you want to. In order to add extra authorities, you would want to edit "validatorCount", "minimumValidatorCount", "invulnerables", "stakers", and "session"."keys"
+
+- Create `genesis.json` from `genesis-spec.json`.
+```
+./polkadot build-spec --chain genesis-spec.json --raw --disable-default-bootnode > genesis.json
+```
+
+- Initiate gossamer node
```

Comment:
nit maybe move that after the `Run Polkadot` step so it's 2 Gossamer steps together?